pretix_eth/network/networks.py

```diff
@@ -3,6 +3,33 @@
 ETH = "ETH"
 DAI = "DAI"
 
+def make_erc_681_url(
+    to_address, payment_amount, chain_id=1,
+    is_token=False, token_address=None
+):
```

Comment:
There is a python code-style standard called PEP8 and I think it is described there how to format multiline function declaration. You can even use an automated tool for formatting called `black`, feel free to check it out> https://pypi.org/project/black/